src/ContactPage/Contact.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import "./Contact.scss";
+
+class Contact extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      name: "",
+      email: "",
+      subject: "",
+      message: "",
+      errors: {
+        name: "",
+        email: "",
+        subject: "",
+        message: ""
+      }
+    };
+  }
+
+  handleChange = event => {
+    event.preventDefault();
+    const validEmailRegex = RegExp(
+      /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i
+    );
+
+    const { name, value } = event.target;
+    let errors = this.state.errors;
```

Comment:
Does this need to be let? I think you can change property values even for const (but double check that)